src/Servers/Kestrel/Transport.Libuv/src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.csproj

```diff
@@ -25,6 +25,7 @@
     <Reference Include="Libuv" />
     <Reference Include="Microsoft.AspNetCore.Hosting.Abstractions" />
     <Reference Include="Microsoft.AspNetCore.Connections.Abstractions" />
+    <Reference Include="Microsoft.AspNetCore.Server.Kestrel.Core" />
```

Comment:
? This creates a circular dependency, what was it for?

Reply:
Libuv build fails without it:

> C:\code\aspnet\aspnetcore\src\Servers\Kestrel\shared\TransportConnection.Generated.cs(11,43): error CS0234: The type or namespace name 'Core' does not exist in the namespace 'Microsoft.AspNetCore.Server.Kestrel' (are you missing an assembly reference?) [C:\code\aspnet\aspnetcore\src\Servers\Kestrel\Transport.Libuv\src\Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.csproj]

Reply:
Hmm, I didn't realize `ITlsApplicationProtocolFeature` lived in Microsoft.AspNetCore.Server.Kestrel.Core.

Reply:
Revert ITlsApplicationProtocolFeature for now, we can't have a circular dependency like this.

Reply:
Done